docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -214,7 +184,17 @@ Verify the sync by checking the logs:
 
 ## Allowlist IP address
 
-You may need to allowlist one of our IP addresses to enable access to Airbyte:
-- 34.106.109.131
-- 34.106.196.165
-- 34.106.60.246
+Depending on your data residency location, you may need to allowlist the following IP addresses to enable access to Airbyte:
+
+### United States and Airbyte Default
```

Comment:
I wonder if we should be explicit about saying that the US/Default IP addresses are GCP us-west-2 
and the EU ip addresses are are both GCP us-west-2 and AWS eu-west-3 (Paris). 

wdyt @davinchia and @malikdiarra?

Reply:
agree. I think it only helps us to be more specific with the info.